### PR TITLE
Fixed wrapper tests #1323

### PR DESCRIPTION
**Pull request by shubhamjain74**

### CODE REVIEW

**Comment by menshikh-iv** [comment]
Thank you @shubhamjain74, It's a nice start :+1: 
I check this PR with full test run on 2.7 and 3.5

File: setup.py [hunk 1]
```diff
@@ -281,6 +281,7 @@ def finalize_options(self):
         'scipy >= 0.7.0',
         'six >= 1.5.0',
         'smart_open >= 1.2.1',
+        'morfessor==2.0.2alpha4',
```
Comment (piskvorky) [inline setup.py]: -1: why should this be a hard dependency??

Has this been thoroughly vetted, by @tmylk and @gojomo ? Is `morfessor` really a core, indispensable part of gensim now, with all that it entails (we'll have to support its installation problems etc).

File: gensim/models/wrappers/ldamallet.py [hunk 2]
```diff
@@ -190,7 +190,7 @@ def load_word_topics(self):
         if hasattr(self.id2word, 'token2id'):
             word2id = self.id2word.token2id
         else:
-            word2id = dict((v, k) for k, v in iteritems(self.id2word))
+            word2id = dict((v, k) for k, v in iteritems(dict(self.id2word)))
```
Comment (piskvorky) [inline gensim/models/wrappers/ldamallet.py]: What is this `dict` casting this for?

File: gensim/models/wrappers/dtmmodel.py [hunk 3]
```diff
@@ -93,7 +93,7 @@ def __init__(
             lencorpus = sum(1 for _ in corpus)
         if lencorpus == 0:
             raise ValueError("cannot compute DTM over an empty corpus")
-        if model == "fixed" and any([i == 0 for i in [len(text) for text in corpus.get_texts()]]):
+        if model == "fixed" and any([i == 0 for i in [len(text) for text in corpus]]):
```
Comment (piskvorky) [inline gensim/models/wrappers/dtmmodel.py]: No need for the outer brackets `[` and `]` (generator fine).

But the whole construction feels unpythonic and strange -- simple `any(not text for text in corpus)` better?

File: gensim/models/wrappers/ldamallet.py (hunk 2 above)
Comment (menshikh-iv) [inline gensim/models/wrappers/ldamallet.py]: It's for python3 compatibility (if self.id2word is `FakeDict` object, `iteritems` can't work correctly)

File: setup.py (hunk 1 above)
Comment (menshikh-iv) [inline setup.py]: It's dependency for `VarEmbed` wrapper. Maybe need to create several mode for install.

File: gensim/models/wrappers/ldamallet.py (hunk 2 above)
Comment (piskvorky) [inline gensim/models/wrappers/ldamallet.py]: Let's update `utils.revdict` then, and use that consistently. There are probably more places like this in the code.

File: setup.py (hunk 1 above)
Comment (piskvorky) [inline setup.py]: Definitely. See the optional `extras_require` setup param.

**Pull request merged**